tests/backstop/config.js

```diff
@@ -32,6 +32,9 @@ module.exports = {
 	asyncCompareLimit: 50,
 	debug: false,
 	debugWindow: false,
+	// Use a custom command template to make sure it works correctly in the GitHub actions environment.
+	// The only difference between the original dockerCommandTemplate and this one is that there is no -t flag in the currect template.
```

Comment:
@aaemnnosttv take a look at the backstopjs [documentation](https://github.com/garris/BackstopJS#using-docker-for-testing-across-different-environments):

> Tip: to run BackstopJS in Docker in an environment where the output is piped (e.g. CI server or an IDE's output window), remove the -t parameter (change the default to "docker run --rm -i --mount...)

If we don't remove `-t`, then the job will fail with exit code `1` because it doesn't support TTY and can't allocate a pseudo-tty. It also answers your question of why we don't have colors.

Reply:
Thanks for clarifying. I wonder what the difference is with Travis since they both ran it in Docker? 🤔